src/scheduler/workers/unpublish/day/sendUnpublishProviderConfirmation.ts

```diff
@@ -0,0 +1,67 @@
+import { ListItemJsonData } from "server/models/listItem/providers/deserialisers/types";
```

Comment:
Looks like this file `sendUnPupublishProviderConfirmation` and `sendUnpubishPostConfirmation` are quite similar. This isn't necessarily a bad thing but if a third file is created that shares these similarities it might make sense to put some of the common in a separate file.